### PR TITLE
[hotfix] Update deprecated parameters in WebOptions 

### PR DESCRIPTION
**Pull request by maqingxiang**
## What is the purpose of the change

Update the deprecated parameters in the WebOptions

### CODE REVIEW

File: flink-core/src/test/java/org/apache/flink/configuration/RestOptionsTest.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public void testBindAddressFirstDeprecatedKey() {
 	public void testBindAddressSecondDeprecatedKey() {
 		final Configuration configuration = new Configuration();
 		final String expectedAddress = "foobar";
-		configuration.setString("jobmanager.web.address", expectedAddress);
```
Comment (zentol) [inline flink-core/src/test/java/org/apache/flink/configuration/RestOptionsTest.java]: this specifically tests that the deprecated key remains functional and must be revert.

File: flink-jepsen/src/jepsen/flink/db.clj [hunk 2]
```diff
@@ -51,7 +51,7 @@
    :high-availability.zookeeper.quorum (zookeeper-quorum test)
    :high-availability.storageDir       (str (:ha-storage-dir test) "/ha")
    :state.savepoints.dir               (str (:ha-storage-dir test) "/savepoints")
-   :web.port                           8081
+   :rest.port                           8081
```
Comment (zentol) [inline flink-jepsen/src/jepsen/flink/db.clj]: values are no longer aligned properly

File: flink-core/src/test/java/org/apache/flink/configuration/RestOptionsTest.java (hunk 1 above)
Comment (maqingxiang) [reply flink-core/src/test/java/org/apache/flink/configuration/RestOptionsTest.java]: Sorry, I was careless. It's been recovered.

File: flink-jepsen/src/jepsen/flink/db.clj (hunk 2 above)
Comment (maqingxiang) [reply flink-jepsen/src/jepsen/flink/db.clj]: ok, It's been changed.

**Pull request merged**